libensemble/tests/scaling_tests/forces/forces_simple_with_input_file/forces_simf.py

```diff
@@ -0,0 +1,64 @@
+import numpy as np
+import jinja2
+
+# Optional status codes to display in libE_stats.txt for each gen or sim
+from libensemble.message_numbers import WORKER_DONE, TASK_FAILED
+
+
+def set_input_file_params(H, sim_specs, ints=False):
+    """
+    This is a general function to parameterize an input file with any inputs
+    from sim_specs["in"]
+
+    Often sim_specs_in["x"] may be multi-dimensional, where each dimension
+    corresponds to a different input name in sim_specs["user"]["input_names"]).
+    Effectively an unpacking of "x"
+
+    """
+    input_file = sim_specs["user"]["input_filename"]
+    input_values = {}
+    for i, name in enumerate(sim_specs["user"]["input_names"]):
+        value = int(H["x"][0][i]) if ints else  H["x"][0][i]
+        input_values[name] = value
+    with open(input_file, "r") as f:
+        template = jinja2.Template(f.read())
+    with open(input_file, "w") as f:
+        f.write(template.render(input_values))
+
+
+def run_forces(H, persis_info, sim_specs, libE_info):
+    """Runs the forces MPI application reading input from file"""
+
+    calc_status = 0
+
+    input_file = sim_specs["user"]["input_filename"]
```

Comment:
This variable (`input_file`) is set but not used in this function.

Reply:
This is resolved in https://github.com/Libensemble/libensemble/pull/1338. I approve this PR